CHANGELOG.md

```diff
@@ -6,6 +6,27 @@ and this library adheres to Rust's notion of
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- **Breaking change:** removed the constants `COMPACT_NOTE_SIZE`,
```

Comment:
Should be in `### Removed`.

Reply:
I was going to address all of this in a subsequent PR after getting #2 in, but I guess now that I've been forced to open a separate PR for just that, I can fix the changelog at the same time.

Reply:
Eh, actually I will still do the changelog rework in a separate PR.